gvsbuild/projects/webrtc_audio_processing.py

```diff
@@ -10,9 +10,9 @@ def __init__(self):
             self,
             "webrtc-audio-processing",
             repository="https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/",
-            version="1.3",
+            version="2.0",
```

Comment:
This is a major bump... Since this is a GStreamer dependency are we sure that the new GStreamer supports this?

Reply:
Hi @pbor, the [NEW](https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/blob/master/NEWS?ref_type=heads) says that their is minor breaking API changes upstream to WebRTC M131. It compiled locally, not sure what impact this has on GStreamer.

@KyleSiefring, any input if we are safe to merge this update to webrtc-audio-processing?

Reply:
It compiled locally, but did GStreamer build all the plugins or it skipped the plugins depending on this lib?

https://gitlab.freedesktop.org/gstreamer/cerbero/-/blob/main/recipes/webrtc-audio-processing.recipe?ref_type=heads still sticks to 1.3 so I am tempted to wait on this one.

Reply:
Hi @pbor, great point, I dropped the webrtc-audio-processing update from the PR and we can bring it back in once Gst has a chance to update.